py-rse/ci.Rmd

```diff
@@ -287,47 +251,49 @@ after_success:
 - codecov
 ```
 
+We use the Travis setting `after_success` since the coverage tool is a separate
+service and the Travis build should happen first.
 We use [`codecov`][codecov] rather than the `coverage` tool described in Section \@ref(py-rse-correct-coverage)
-because it is a service like Travis CI.
+since it is a service like Travis CI while the `coverage` tool is a program.
```

Comment:
I don't think this distinction will be clear to our intended readers at this point.